core/src/main/java/tc/oc/pgm/community/commands/ReportCommands.java

```diff
@@ -84,6 +84,15 @@ public void report(MatchPlayer sender, Match match, Player player, @Text String
     }
 
     MatchPlayer accused = match.getPlayer(player);
+
+    // Don't allow reports of vanished players
+    if (accused.isVanished()) {
+      sender.sendWarning(
+          TextComponent.of(
+              "Could not find player named '" + player.getName() + "'", TextColor.RED));
```

Comment:
Let me know if you want this translated. I was unsure since the matching text error comes from Intake, I did not believe it would matter since other languages most likely see the English warning no matter what. 

See [here](https://github.com/Electroid/intake/blob/e34dfe29eb0f79bde70dc8678c55ea9aafd568e0/bukkit/src/main/java/app/ashcon/intake/bukkit/parametric/provider/DynamicPlayerProvider.java#L44) for details.

Reply:
Yes, add a note.